bridges/snowbridge/primitives/outbound-queue/src/v2/converter/convert.rs

```diff
@@ -174,8 +174,7 @@ where
 
 			// Ensure PNA already registered
 			let token_id = TokenIdOf::convert_location(&asset_id).ok_or(InvalidAsset)?;
-			let expected_asset_id = ConvertAssetId::convert(&token_id).ok_or(InvalidAsset)?;
-			ensure!(asset_id == expected_asset_id, InvalidAsset);
```

Comment:
@franciscoaguirre are there perhaps other ways to check asset ID equivalence, across different XCM versions?

Reply:
Actually, I assume the check here is unnecessary, as the conversion alone is sufficient to verify whether the token is registered.